Projects/hGraphQL/Sources/TokenRefresher.swift

```diff
@@ -16,11 +15,14 @@ public class TokenRefresher {
         return Date().addingTimeInterval(60) > token.accessTokenExpirationDate
     }
 
+    public func getAccessToken() -> String? {
```

Comment:
Seems like you never ended up using this one?

Reply:
Removed 😅